sprimber-spring-boot-autoconfigure/pom.xml

```diff
@@ -34,6 +34,14 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>com.griddynamics.qa</groupId>
+            <artifactId>sprimber-cucumber3</artifactId>
```

Comment:
With that you have a dependency from top module to bottom - which is not correct. It brakes the idea of your abstraction. Please find a way to avoid it.
Even if you made it just for test cases - it's not correct.
Test cases scope should not depend on any foreign implementation.
You can prepare your own implementation for test cases and keep it up to date along with core functionality changes.